usd/src/cmd/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+if(TARGET ${usd_target})
+  add_executable(sdf2usd
+    sdf2usd.cc
+  )
+
+  target_link_libraries(sdf2usd
+    PUBLIC
+      ignition-utils${IGN_UTILS_VER}::ignition-utils${IGN_UTILS_VER}
+      ignition-common${IGN_COMMON_VER}::ignition-common${IGN_COMMON_VER}
+      ignition-common${IGN_COMMON_VER}::graphics
```

Comment:
if we are linking this target against the `usd_target`, then I don't think we need these lines here because `usd_target` has already linked against them publicly, right?

Reply:
I think we can remove the indirect dependencies (eg. `ignition-common${IGN_COMMON_VER}::graphics`) and keep the direct ones (`ignition-utils${IGN_UTILS_VER}::ignition-utils${IGN_UTILS_VER}`)